packages/mobile/src/identity/verification.ts

```diff
@@ -64,6 +68,13 @@ export interface AttestationCode {
   issuer: string
 }
 
+export function* isSufficientBalance() {
+  const e164Number: string = yield select(e164NumberSelector)
+  const cachedSalt = yield call(getSaltCache, e164Number)
+  const isSufficientForQuota = cachedSalt ? true : yield call(balanceSufficientForQuotaRetrieval)
+  yield put(setVerificationPossible(isSufficientForQuota))
+}
```

Comment:
I feel we shouldn't introduce an action and state for this.
If it can be derived from the existing state, we should derive it directly and use a selector.

Reply:
I've tried to use `verificationStatus` for this, but then in the situation when we need to change it by the reasons unrelated to quota - it got mixed up, so decided to have it as a separate flag. It felt for me not as a status, but rather a precondition. Do you have a better idea on how we can derive it?

Reply:
From what I see, this logic could be done in a selector instead.
Happy to have a deeper look at this together (slack or call).